examples/sse.js

```diff
@@ -0,0 +1,23 @@
+import sse from "k6/experimental/sse";
+import {check} from "k6";
+
+export default function () {
+	var url = "https://echo.websocket.org/.sse";
+	var params = {"tags": {"my_tag": "hello"}};
+
+	var response = sse.open(url, params, function (client) {
```

Comment:
An SSE connection is usually long-lived. We do not test normal operation by opening a new connection in each iteration.

Reply:
Not at all, see: https://platform.openai.com/docs/api-reference/streaming

I mainly use K6 at the moment to test LLM inference server:
 - https://github.com/ggerganov/llama.cpp/pull/5941